lacework/resource_lacework_resource_group_gcp.go

```diff
@@ -0,0 +1,200 @@
+package lacework
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+	"strings"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkResourceGroupGcp() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkResourceGroupGcpCreate,
+		Read:   resourceLaceworkResourceGroupGcpRead,
+		Update: resourceLaceworkResourceGroupGcpUpdate,
+		Delete: resourceLaceworkResourceGroupGcpDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkResourceGroup,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The resource group name",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "The state of the resource group",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The description of the resource group",
+			},
+			"organization": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The GCP organization",
+			},
+			"projects": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(val.(string))
+					},
+				},
+				Required:    true,
+				Description: "The list of GCP projects to include in the resource group",
+			},
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The resource group unique identifier",
+			},
+			"lacework_account_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The lacework account id",
+			},
+			"last_updated": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The time in millis when the resource was last updated",
```

Comment:
I just noticed that we are not setting the actual time but the milliseconds, sorry for asking this soooo late but, why we didn't convert this to an actual Date/Time?

Reply:
I'll put in a fix to the go-sdk to resolve the date/time